### PR TITLE
Update README.md to include instructions for using test utils globally

### PR DESCRIPTION
**Pull request by kylemh**
Took me like 5 minutes to parse issues and find the clear instructions from one your comments, so I thought this would be helpful!

### CODE REVIEW

**Comment by thebuilder** [comment]
Cool - I dont think adding it the setup should be the default, unless your codebase depends on the Intersection Observer globally. 

Importing the utils inside a test file, ensures it's created when you need to interact with it.

After adding `fallbackInView` support, that could be used as an alternative to remove the requirement for Intersection Observer to exist, since it will skip throwing an error if you set it.

**Comment by kylemh** [comment]
While not global, I've had a few companies which want analytics reported when an element is _seen_. It happens so much that simply use the global utils and then use the imported methods when I'm actually testing the intersection behavior.

**Review by thebuilder — COMMENTED** [review]
If we can add a line about why you'd import it globally (To ensure the Intersection Observer is always mocked), and that it's optional.

Switch the order, so "Global" is placed after the "Per file" - It's not commonly needed. Maybe lose the "Per file" header, and change "Global" to "Always Mock IntersectionObserver" or similar.

Suggest the [unsupported fallback](https://github.com/thebuilder/react-intersection-observer#unsupported-fallback) as alternative, if you don't need to mock it?

**Comment by kylemh** [comment]
Before I commit around the unsupported fallback aspect of your comment, I want to make sure I get it right.

The suggestion _there_ I believe would be that you would like a normal setupFile for jest which simply calls `defaultFallbackInView(true);`.

**Comment by thebuilder** [comment]
> Before I commit around the unsupported fallback aspect of your comment, I want to make sure I get it right.
> 
> Are you suggesting to use a normal `setupFile` for jest which simply calls `defaultFallbackInView(true);`.

Yeah, calling it with either `true` or `false` would work, depending on what makes most sense in a given setup.

**Comment by kylemh** [comment]
Sorted!

File: README.md [hunk 1]
```diff
@@ -267,6 +271,32 @@ Call the `intersectionMockInstance` method with an element, to get the (mocked)
 `IntersectionObserver` instance. You can use this to spy on the `observe` and
 `unobserve` methods.
 
+#### Global Intersection Observer Behavior
+
+##### Use Fallback
+
+You can create a [Jest setup file](https://jestjs.io/docs/configuration#setupfiles-array) that leverages the [unsupported fallback](https://github.com/thebuilder/react-intersection-observer#unsupported-fallback) to that tests encountering Intersection Observer all behave the same way, like so:
```
Comment (thebuilder) [inline README.md:278]:
```suggestion
You can create a [Jest setup file](https://jestjs.io/docs/configuration#setupfiles-array) that leverages the [unsupported fallback](https://github.com/thebuilder/react-intersection-observer#unsupported-fallback) option. In this case, you only mock the IntersectionObserver in test files were you actively import `react-intersection-observer/test-utils`:
```

Comment (thebuilder) [reply README.md:278]: Last part of the sentence is missing a word or two- maybe something more like this?

File: README.md [hunk 2]
```diff
@@ -267,6 +271,32 @@ Call the `intersectionMockInstance` method with an element, to get the (mocked)
...
+import { defaultFallbackInView } from 'react-intersection-observer';
+
+defaultFallbackInView(true); // or 'false' - whichever consistent behavior makes the most sense for your use case.
+```
+
+##### Mock Everywhere
+
+In your Jest config, add `'react-intersection-observer/test-utils'` to the array value for the [`setupFilesAfterEnv`](https://jestjs.io/docs/configuration#setupfilesafterenv-array) option.
+
+Like so:
```
Comment (thebuilder) [inline README.md:290]:
```suggestion

```

**Review by thebuilder — APPROVED** [review]

**Comment by thebuilder** [comment]
Thanks for the help 🙏

**Pull request merged**